docs/html/annotated.html

```diff
@@ -24,7 +24,7 @@
  <tr style="height: 56px;">
```

Comment:
Do you really want to check in all the generated html files?

Reply:
It needs to be AFAIK in order to host the html.

Reply:
But then a pull of the repo gets all the HTML. Generally we separate generated docs from docs source for that reason. e.g. cuDF does not ship the generated HTML docs, we host them on the rapids.ai site. https://github.com/rapidsai/cudf/tree/branch-0.13/docs/cudf

Reply:
Yeah, I don't have the infrastructure set up to do that :) 

This was just a temporary solution to allow me to easily host the Doxygen HTML.

Reply:
Okay, I managed to get the docs removed from showing up on this PR. They are still committed to the master branch, but that's only so I can leverage github hosting.

Reply:
I wouldn't  host the docs at all, if I were you. Once it moves to NVIDIA org, then we can figure that out. I don't think it's friendly to the user to have to pull down all the HTML when they clone.

Reply:
Since this repo is temporary, I wanted to host the docs so people could see what they look like. In the final home, they won't be part of the repo.